src/Components/VotesCount.jsx

```diff
@@ -0,0 +1,43 @@
+import { useState, useEffect } from "react";
+import { useParams } from "react-router-dom";
+import { getVoteCount } from "../api";
+
+function VotesCount() {
+  const { article_id } = useParams();
+  const [voteCount, setVoteCount] = useState([]);
+  const [isError, setIsError] = useState(null);
+
+  useEffect(() => {
+    getVoteCount(article_id).then((voteCount) => {
+      setVoteCount(voteCount);
+    });
+  }, []);
+
+  const handleAddVote = () => {
+    setIsError(null);
+    setVoteCount((currentVoteCount) => currentVoteCount + 1);
+    if (isError) {
+      setVoteCount((currentVoteCount) => currentVoteCount - 1);
+      setIsError("Your vote was not successful. Please try again!");
+    }
+  };
+
+  const handleSubtractVote = () => {
+    setVoteCount((currentVoteCount) => currentVoteCount - 1);
+    if (isError) {
+      setVoteCount((currentVoteCount) => currentVoteCount + 1);
+      setIsError("Could not rescind your vote. Please try again!");
+    }
+  };
```

Comment:
I'd consider adding in a .catch for your error handling. You want to make sure that the error is caught in the API call. 💡